src/ncp/spinel_transformer.hpp

```diff
@@ -0,0 +1,35 @@
+/**
+ * NCP Spinel Transformer allows to transform spinel frames sent between Application Processor (AP) and Network Co-Processor (NCP).
+ */
+
+namespace SpinelTransformer {
```

Comment:
The name of this class is pretty generic, but the stated use case is pretty specific. Are there other types of use cases you imagine this hook being used for?

Reply:
Not many solid use cases apart from encryption. Maybe some logging, stats or sniffing. Maybe compression as well (if CPUs are strong, and link bandwidth is slow/limited).

Reply:
Can we then rename it to make it clear that the intention is for this class to be used link security?

Reply:
SpinelEncrypter?